test/test_torch.py

```diff
@@ -6124,14 +5992,9 @@ def scipy_tri_solve_batched(A, B, upper, trans, diag):
             return flat_X.reshape(expand_B.shape)
 
         def run_test(A_dims, b_dims, cast, upper, transpose, unitriangular):
-            A = torch.randn(*A_dims)
-            A = A.triu() if upper else A.tril()
-            if unitriangular:
-                A.diagonal(dim1=-2, dim2=-1).fill_(1.)
-            b = torch.randn(*b_dims)
-            x_exp = torch.Tensor(scipy_tri_solve_batched(A.numpy(), b.numpy(),
+            b, A = triangular_solve_test_helper(A_dims, b_dims, cast, upper, unitriangular)
+            x_exp = torch.Tensor(scipy_tri_solve_batched(A.cpu().numpy(), b.cpu().numpy(),
```

Comment:
Don't use the legacy `torch.Tensor` constructor, use `torch.tensor`

Reply:
I keep forgetting :-(